jenkins/pipelines/ci/tikv/tikv_ghpr_test.groovy

```diff
@@ -102,31 +102,6 @@ try {
                                 }
                                 checkout changelog: false, poll: false, scm: [$class: 'GitSCM', branches: [[name: 'master']], doGenerateSubmoduleConfigurations: false, extensions: [[$class: 'PruneStaleBranch'], [$class: 'CleanBeforeCheckout']], submoduleCfg: [], userRemoteConfigs: [[credentialsId: 'github-sre-bot-ssh', refspec: '+refs/pull/*:refs/remotes/origin/pr/*', url: 'git@github.com:tikv/tikv.git']]]
                             }
-                            dir("/home/jenkins/agent/tikv-${ghprbTargetBranch}/build") {
```

Comment:
I'm afraid disabling lint cache can make compile time even worse.

Reply:
I create another pr https://github.com/PingCAP-QE/ci/pull/184/files to disable incremental build